en/docs/references/ballerina-expression-syntax.md

```diff
@@ -1,12 +1,12 @@
 # Ballerina Expression Syntax
```

Comment:
Shouldn't we change this to `Ballerina expression syntax`?

Reply:
We use title-style capitalization for the page title and sentence-style capitalization for all other subsection headings. Since this is the page title it should be in title-style capitalization.